functions.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/** Query to return all review records for main collection page display
+ *
+ * @param PDO $db
+ *
+ * @return array
+ */
+function dbQueryGetAll(PDO $db) : array {
+    $query = $db->prepare("SELECT `burger_name`, `restaurant`, `visit_date`, `image`, `price`, `patty_rating`, `topping_rating`, `sides_rating`, `value_rating`, `total_score` FROM `reviews`");
+    $query->execute();
+    return $query->fetchAll();
+}
+
+/** Cycles through reviews, changing the date format to dd/mm/yyyy from sql yyyy/mm/dd
+ *
+ * @param array $all_reviews
+ *
+ * @return array
+ */
+function dateFormatUK(array $all_reviews) : array {
+    if ($all_reviews != null) {
```

Comment:
can all_reviews be null at this point? wouldnt the type hinting of the param pick this up?

Reply:
Passing the function an empty array triggers the 'else' statement of this condition, so it seems to serve a purpose!

Reply:
I see, what about using is_null function instead? can make it opposite with !